tests/baselines/reference/accessorDeclarationEmitVisibilityErrors.errors.txt

```diff
@@ -1,12 +1,9 @@
 accessorDeclarationEmitVisibilityErrors.ts(2,18): error TS2304: Cannot find name 'DoesNotExist'.
-accessorDeclarationEmitVisibilityErrors.ts(2,18): error TS4106: Parameter 'arg' of accessor has or is using private name 'DoesNotExist'.
 
 
-==== accessorDeclarationEmitVisibilityErrors.ts (2 errors) ====
+==== accessorDeclarationEmitVisibilityErrors.ts (1 errors) ====
     export class Q {
         set bet(arg: DoesNotExist) {}
                      ~~~~~~~~~~~~
 !!! error TS2304: Cannot find name 'DoesNotExist'.
-                     ~~~~~~~~~~~~
-!!! error TS4106: Parameter 'arg' of accessor has or is using private name 'DoesNotExist'.

```

Comment:
This is unrelated to type assertions, no? Does this PR contain changes for the getter/setter stuff too?

Reply:
The declared accessor type was not visited. Fixed.